ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/MessageScrubberTest.java

```diff
@@ -280,11 +280,17 @@ public void testBug1642386Affected() throws Exception {
     assertFalse(MessageScrubber.bug1642386Affected(ImmutableMap.<String, String>builder()
         .putAll(baseAttributes).put(Attribute.APP_VERSION, "60.6.1").build()));
     assertFalse(MessageScrubber.bug1642386Affected(ImmutableMap.<String, String>builder()
-        .putAll(baseAttributes).put(Attribute.APP_VERSION, "10.25.1").build()));
+        .putAll(baseAttributes).put(Attribute.APP_VERSION, "27.1").build()));
     assertTrue(MessageScrubber.bug1642386Affected(ImmutableMap.<String, String>builder()
         .putAll(baseAttributes).put(Attribute.APP_VERSION, "25.0.1").build()));
     assertTrue(MessageScrubber.bug1642386Affected(ImmutableMap.<String, String>builder()
         .putAll(baseAttributes).put(Attribute.APP_VERSION, "26.1").build()));
+    assertTrue(MessageScrubber.bug1642386Affected(ImmutableMap.<String, String>builder()
+        .putAll(baseAttributes).put(Attribute.APP_VERSION, "14.1").build()));
+    assertTrue(MessageScrubber.bug1642386Affected(ImmutableMap.<String, String>builder()
+        .putAll(baseAttributes).put(Attribute.APP_VERSION, "8.0.2").build()));
+    assertTrue(MessageScrubber.bug1642386Affected(ImmutableMap.<String, String>builder()
+        .putAll(baseAttributes).put(Attribute.APP_VERSION, "10.0.2").build()));
```

Comment:
Can you add a test for Firefox 100.0.0? It looks like the regex should be fine, but its good to check.